ingeniamotion/configuration.py

```diff
@@ -826,19 +826,27 @@ def is_sto_inactive(self, servo: str = DEFAULT_SERVO, axis: int = DEFAULT_AXIS)
         """
         return self.get_sto_status(servo, axis) == self.STO_INACTIVE_STATE
 
-    def is_sto_abnormal_latched(self, servo: str = DEFAULT_SERVO, axis: int = DEFAULT_AXIS) -> bool:
+    def is_sto_abnormal_latched(
+        self, servo: str = DEFAULT_SERVO, axis: int = DEFAULT_AXIS
+    ) -> SeverityLevel:
```

Comment:
In my opinion, we should create a new enum instead of reuse a existing one.
We can use IntEnum and set the abnormal to 1 and normal to 0, to be a bit compatible with old version (but not totally because in old version returns a float).